### PR TITLE
Added settings for adding Google Analytics tracking code

### PR DESCRIPTION
**Pull request by antonioperic**

### CODE REVIEW

**Comment by antonioperic** [comment]
@loevstroem yes that for production sounds good. And regarding multishops need to wait to see complete PR, but for sure it will be needed to have key for every single shop in multi shop setup

File: src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php [hunk 1]
```diff
@@ -67,38 +75,64 @@ public function buildSettings(SettingsBuilderInterface $builder)
     public function buildForm(FormBuilderInterface $builder)
     {
         $builder
-            ->add('title', 'text', array(
-                'label'       => 'sylius.form.settings.general.title',
-                'constraints' => array(
-                    new NotBlank()
+            ->add(
```
Comment (aRn0D) [inline src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php]: should be  `->add('title', 'text', array( ... (same line)`

File: src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php [hunk 2]
```diff
@@ -67,38 +75,64 @@ public function buildSettings(SettingsBuilderInterface $builder)
...
-            ))
-            ->add('currency', 'sylius_currency_choice', array(
-                'label'       => 'sylius.form.settings.general.currency',
-                'constraints' => array(
-                    new NotBlank(),
-                    new Currency(),
+            )
+            ->add(
+                'google_analytics_code',
+                'text',
```
Comment (aRn0D) [inline src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php]: The text form type but you seem to use it as boolean in your template. do I miss something ?

Comment (aRn0D) [inline src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php]: Perhaps you can transform it in textarea and let's the use put its owns script, what do you thing ?

**Comment by antonioperic** [comment]
@aRn0D I think text is better option as some of users are suing GTM some analytics some other things

File: src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php (hunk 1 above)
Comment (antonioperic) [inline src/Sylius/Bundle/CoreBundle/Settings/GeneralSettingsSchema.php]: I don't think it is so important, as both ways is standards. but i will rework in way that is already in file

**Comment by antonioperic** [comment]
@aRn0D added fixes, please give feedback

**Comment by antonioperic** [comment]
ping @aRn0D

**Comment by aRn0D** [comment]
@antonioperic The build failed!

@Sylius/core-team What do you think? For me, it can be merge when the build will be green.

**Comment by umpirsky** [comment]
:+1:

**Comment by antonioperic** [comment]
@aRn0D  it should be green now :)

**Comment by pjedrzejewski** [comment]
:+1:

**Comment by antonioperic** [comment]
@aRn0D  all green :)

**Comment by aRn0D** [comment]
Thank you Antonio.

**Pull request merged**